doc/source/user_guide/visualization.rst

```diff
@@ -1026,7 +1026,7 @@ unit interval). The point in the plane, where our sample settles to (where the
 forces acting on our sample are at an equilibrium) is where a dot representing
 our sample will be drawn. Depending on which class that sample belongs it will
 be colored differently.
-See the R package `Radviz <https://cran.r-project.org/web/packages/Radviz/>`__
+See the R package `Radviz <https://cran.r-project.org/package=Radviz/>`__
```

Comment:
So when pasting this in my browser it redirects to the URL prior to this commit - is there any reason for changing this?

Reply:
At the bottom of the page it says:

>Please use the canonical form https://CRAN.R-project.org/package=Radviz to link to this page.

It's a little strange that it redirects back to the original URL but seems to be CRAN's preference to use those links.

Reply:
Hmm that's weird. Makes sense for this PR then!